packages/fxa-payments-server/package.json

```diff
@@ -8,7 +8,9 @@
     "lint:sass": "sass-lint -v",
     "lint:deps": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
     "start": "node server/bin/fxa-payments-server.js",
-    "start-dev": "concurrently -k \"PROXY_STATIC_RESOURCES_FROM='http://127.0.0.1:3032' node server/bin/fxa-payments-server.js\" \"react-scripts start\"",
```

Comment:
Slight tweak to the scripts here to use `npm-run-all --parallel` and drop the dependency on `concurrently`